rosbag2/include/rosbag2/sequential_reader.hpp

```diff
@@ -79,6 +80,23 @@ class ROSBAG2_PUBLIC SequentialReader
    */
   virtual bool has_next();
 
+    /**
+   * Ask whether there is another database file to read from the list of relative
+   * file paths.
+   *
+   * \return true if there are still files to read in the list
+   */
+  virtual bool has_next_file();
+
+  /**
+   * Update the current database file to read from and remove the first (front)
+   * element in the list of files to read from.
+   *
+   * Expected usage:
+   * if (has_next_file()) get_next_file();
```

Comment:
please document what will happen if has_next_file() is false and this function is called.